.tekton/scripts/check-partner-tasks.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPTDIR/../.." || exit 1
+
+# Expected partner task directory structure:
+# task/partners/<task name>/<version>/<task file>
+
+if [ ! -e "partners/" ]; then
```

Comment:
Do you think naming the directory to `partner_task` would be a good idea instead of `partners`?

Reply:
I would use `partners`. Word "task" inside `partner_task` looks redundant.